parts/dcoscustomdata190.t

```diff
@@ -303,3 +304,42 @@ write_files:
   content: 'ATTRIBUTES_STR'
   permissions: "0644"
   owner: "root"
+- content: |
+    #!/bin/bash
+    # Check the partitions on /dev/sdb created by cloudinit and force a detach and
+    # reformat of the parition.  After which, all will be remounted.
+    EPHEMERAL_DISK="/dev/sdb"
+    PARTITIONS=`fdisk -l $EPHEMERAL_DISK | grep "^$EPHEMERAL_DISK" | cut -d" " -f1 | sed "s~$EPHEMERAL_DISK~~"`
+    if [ -n "$PARTITIONS" ]; then
+        for f in $PARTITIONS; do
+            df -k | grep "/dev/sdb$f"
+            if [ $? -eq 0 ]; then
+                umount -f /dev/sdb$f
+            fi
+            mkfs.ext4 /dev/sdb$f

```

Comment:
I noticed no real additional latency, this is on the ephemeral disk which is already partitioned.  I ran this on smaller DS* instances as well as larger L* instances and no real difference.

One reason where "lazy format" may not be the best option here is that the ephemeral disk partitions are used for mesos and docker data.  It works right now because one partition is accurately formatted by cloud-init, the others aren't and just default to the OS disk.